docs/django.md

```diff
@@ -10,126 +10,176 @@ weight=4
+5. Create a `requirements.txt` in your project directory.
 
-    $ docker-compose run web django-admin.py startproject composeexample .
+    This file is used by the `RUN pip install -r requirements.txt` command in your `Dockerfile`.
 
-First, Compose will build an image for the `web` service using the `Dockerfile`.
-It will then run `django-admin.py startproject composeexample .` inside a
-container built using that image.
+6. Add the required software in the file.
 
-This will generate a Django app inside the current directory:
+        Django
+        psycopg2
 
-    $ ls
-    Dockerfile       docker-compose.yml          composeexample       manage.py        requirements.txt
+7. Save and close the `requirements.txt` file.
 
-### Connect the database
+8. Create a file called `docker-compose.yml` in your project directory.
 
-Now you need to set up the database connection. Replace the `DATABASES = ...`
-definition in `composeexample/settings.py` to read:
+    The `docker-compose.yml` file describes the services that make your app. In
+    this example those services are a web server and database.  The compose file
+    also describes which Docker images these services use, how they link
+    together, any volumes they might need mounted inside the containers.
+    Finally, the `docker-compose.yml` file describes which ports these services
+    expose. See the [`docker-compose.yml` reference](yml.md) for more
+    information on how this file works.
 
-    DATABASES = {
-        'default': {
-            'ENGINE': 'django.db.backends.postgresql_psycopg2',
-            'NAME': 'postgres',
-            'USER': 'postgres',
-            'HOST': 'db',
-            'PORT': 5432,
-        }
-    }
+9. Add the following configuration to the file.
+
+        db:
+          image: postgres
+        web:
+          build: .
+          command: python manage.py runserver 0.0.0.0:8000
+          volumes:
+            - .:/code
+          ports:
+            - "8000:8000"
+          links:
+            - db
+
+    This file defines two services. The `db` service and the `web` service.
+
+10. Save and close the `docker-compose.yml` file.
+
+## Create a Django project
+
+In this step, you create a Django started project by building the image from the build context defined in the previous procedure.
+
+1. Change to the root of your project directory.
+
+2. Create the Django project using the `docker-compose` command.
+
+        $ docker-compose run web django-admin.py startproject composeexample .
+
+    This instructions Compose to the run `django-admin.py startproject
+    composeexample` in a container. Before it can do this, Compose needs to know
+    what to build and it looks in the current directory for instructions. The
+    `docker-compose.yml` defines two services, the `web` service defines a `build`
+    context as the current (.) directory. Finding the Dockerfile, Compose uses
+    that to build the `web` service.
```

Comment:
This bit reads strangely to me. How about:

> This instructs Compose to run `django-admin.py startproject composeeexample` in a container, using the `web` service's image and configuration. Because the image doesn't exist yet, Compose builds it from the current directory, as specified by the `build: .` line in `docker-compose.yml`.